frontend/app/routes/_gcweb-app.personal-information.address.confirm.tsx

```diff
@@ -23,20 +27,20 @@ export async function action({ request }: ActionFunctionArgs) {
 
 export default function ConfirmAddress() {
   const loaderData = useLoaderData<typeof loader>();
-
+  const { t } = useTranslation(i18nNamespaces);
   return (
     <>
       <h1 id="wb-cont" property="name">
-        Change address
+      {t('personal-information:edit.change-address')}
       </h1>
-      <h2 className="h3">Confirm</h2>
+      <h2 className="h3">{t('gcweb:input-label.button.confirm')}</h2>
```

Comment:
As alluded to in a previous comment, even though `Confirm` might be a common button label across the application, we should still define it again in this view's own translation file.

Years ago I worked on an application where they wanted to change the label of a cancel button to "go back" and the developer made the change and it altered 15+ pages in the application (and they had to roll back).

Reply:
I agree with @gregory-j-baker. From my experience the confirm or action text should be specific for his context.